npdarrington/instaquizzes
# Nd bug question html

### What’s this PR do?  
- Uses dangerouslySetInnerHTML when displaying questions to filter our raw HTML.
 
### Where should the reviewer start?  
- git pull this branch.
 
### How should this be manually tested?  
- run `npm start` and navigate to localhost:3000
- modify state to display a question that will display raw HTML and ensure the HTML is converted instead of displaying raw data.
 
### Any background context you want to provide?  
- Completely forgot about dangerouslySetInnerHTML and made a whole function just to reverse it. Oops!.
 
### What are the relevant tickets?  
- closes #21
